src/pilot/deploy-overcloud.py

```diff
@@ -663,18 +310,12 @@ def main():
                             action='store_true',
                             help="Indicates if the deploy-overcloud script "
                                  "should be run in debug mode")
-
-        LoggingHelper.add_argument(parser)
```

Comment:
Removing the above line breaks the ability to pass in a command line argument to set the logging level.

Reply:
Required Added